config.ini

```diff
@@ -4,7 +4,9 @@
 github-server = github.com
 # Leave empty to not have a secret.
 github-hook-secret =
-github-token = a-token
+github-token =
+# chevah-robot
+#github-token = 368ad3be0624a1e04b47406219772ff5babbe6a9
```

Comment:
Got an automated alert from GitHub when this token committed on Friday. 

Consequently, it was automatically revoked because it was included in a commit.

Reply:
Wow. I am ashamed to have made such a blunder.

It looks like chevah-robot is down now (as the token was revoked). I will think about my process and talk with Adi.

Reply:
Take it easy! To err is humane… ;-]

Let me know if there's anything left to help with.

Reply:
> To err is humane… ;-]

Indeed. To really mess things up you need a computer! :laughing:

> Let me know

Nope, thank you!

I am splitting the secrets from the settings file, then will deploy the bot with the new token in order to test this branch as well (two birds with one stone).